### PR TITLE
TC_005.002.002 | Main page > Cart > Proceed to checkout with valid data

### PR DESCRIPTION
**Pull request by vguliakin**
https://trello.com/c/qiYD156D/210-tc005002002-main-page-cart-proceed-to-checkout-with-valid-data

### CODE REVIEW

**Comment by vguliakin** [comment]
[TC_005.002.002 | Main page > Cart > Proceed to checkout with valid data](https://trello.com/c/qiYD156D/210-tc005002002-main-page-cart-proceed-to-checkout-with-valid-data)

File: LICENSE
Comment (aqaestuser) [inline LICENSE:1]: Зачем удалять LICENSE ?

File: README.md
Comment (aqaestuser) [inline README.md:1]: Чем README не угодил?

**Review by aqaestuser — CHANGES_REQUESTED** [review]

**Review by aqaestuser — APPROVED** [review]

**Comment by aqaestuser** [comment]
Ладно пусть так, удаленное мы вернем

**Pull request closed without merging**